pkg/controller/workspace/component/plugins_utils.go

```diff
@@ -155,6 +155,18 @@ func convertContainers(pluginContainers []brokerModel.Container, wkspCtx model.W
 		if err != nil {
 			return nil, err
 		}
+
+		if config.ControllerCfg.GetWebhooksEnabled() == "true" {
+			if container.Env == nil {
+				container.Env = []corev1.EnvVar{}
+			}
```

Comment:
You technically don't need to initialize a nil *slice* (unlike a map ;) ) before appending to it

Reply:
:+1: